qiita_db/handlers/processing_job.py

```diff
@@ -0,0 +1,157 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from datetime import datetime
+from json import loads
+
+from tornado.web import RequestHandler
+
+import qiita_db as qdb
+
+
+def _get_job(job_id):
+    """Returns the job with the given id if it exists
+
+    Parameters
+    ----------
+    job_id : str
+        The job id to check
+
+    Returns
+    -------
+    qiita_db.processing_job.ProcessingJob, bool, string
+        The requested job or None
+        Whether if we could get the job or not
+        Error message in case we couldn't get the job
+    """
+    if not qdb.processing_job.ProcessingJob.exists(job_id):
+        return None, False, 'Job does not exist'
+
+    try:
+        job = qdb.processing_job.ProcessingJob(job_id)
+    except qdb.exceptions.QiitaDBError as e:
+        return None, False, 'Error instantiating the job: %s' % str(e)
+
+    return job, True, ''
+
+
+class JobHandler(RequestHandler):
+    def get(self, job_id):
+        """Retrieves the work that need to be done for the job
+
+        Parameters
+        ----------
+        job_id : str
+            The job that the information needs to be returned
+        """
```

Comment:
Yeah, I was thinking about that - I was unsure on how to document the actual results of the handler given that they're not "returning" anything in the usual python way, but they're actually returning data through the HTTP requests. Given that, I think it is ok to document them in the "Returns" section. Will do that!